src/components/ListManager.jsx

```diff
@@ -1,55 +1,56 @@
-var React = require('react');
-var ReactDOM = require('react-dom');
-var List = require('./List.jsx');
+import React from 'react';
+import List from './List.jsx';
 
-var ListManager = React.createClass({
-  getInitialState: function() {
-    return {items: [], newItemText: ''};
-  },
-  onChange: function(e) {
+class ListManager extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {items: [], newItemText: ''};
+    this.onChange = this.onChange(this);
+    this.handleSubmit = this.handleSubmit(this);
```

Comment:
this.onChange() and this.handleSubmit() are getting invoked here, hence throwing a javascript error. I think you meant to bind the instance instead.

Code should be updated to:
```
this.onChange = this.onChange.bind(this);
this.handleSubmit = this.handleSubmit.bind(this);
```

Reply:
I added a .babelrc with transform-class-propeprties and changed all handlers to arrow functions to avoid .bind(this) requirement. Also removed extraneous constructors....

Reply:
Great! I've just started delving into the world of Babel and ES6. Thanks.